services/workflows-service/src/data-analytics/types.ts

```diff
@@ -19,3 +25,11 @@ export type TransactionsAgainstDynamicRulesType = {
   groupByBusiness?: boolean;
   groupByCounterparty?: boolean;
 };
+
+export type AnyArray = any[];
+
+export type AnyRecord = Record<PropertyKey, any>;
+
+export type GenericFunction = (...args: AnyArray) => any;
+
+export type GenericAsyncFunction = (...args: AnyArray) => Promise<any>;
```

Comment:
Let's move these types to `workflows-service/src/types.ts` for now. I'll add a tech debt task to have some way to share these sort of types in our system.